src/core/types/app_categories.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/** @public */
+export enum AppCategory {
```

Comment:
It seems this type is not used on the server. Any reason this can't exist in `src/core/public/application/types`? Also this file should be named `app_category.ts`

Reply:
It's used in legacy server code actually

Reply:
See `x-pack/legacy/plugins/siem/index.ts` for exemple

Reply:
Right, but this is legacy server files.

Assuming there's no issues importing from `src/core/public` inside legacy server code, I think we should design this with NP plugins in mind, where this will never be imported from server modules.

Reply:
Good point. then @myasonik I apologize about asking for this change.